### PR TITLE
Replacing the unmaintained gulp-livereload with gulp-refresh

### PR DESCRIPTION
**Pull request by lirantal**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,13 +88,13 @@
...
     "gulp-istanbul": "~1.1.1",
     "gulp-less": "~3.1.0",
-    "gulp-livereload": "~3.8.1",
     "gulp-load-plugins": "~1.3.0",
     "gulp-mocha": "~3.0.1",
     "gulp-ng-annotate": "~2.0.0",
     "gulp-node-inspector": "~0.1.0",
     "gulp-nodemon": "~2.2.1",
     "gulp-protractor": "~3.0.0",
+    "gulp-refresh": "^1.1.0",
```
Comment (simison) [inline package.json]: `~` instead of `^`

Comment (lirantal) [inline package.json]: Right

**Comment by mleanos** [comment]
LGTM

**Comment by lirantal** [comment]
Thanks guys

**Pull request merged**